src/Datadog.Trace/Tags.cs

```diff
@@ -405,5 +405,13 @@ public static class Tags
         internal const string ElasticsearchUrl = "elasticsearch.url";
 
         internal const string RuntimeId = "runtime-id";
+
+        internal const string AzureFunctionTriggerType = "aas.function.trigger";
+
+        internal const string AzureFunctionName = "aas.function.name";
+
+        internal const string AzureFunctionMethod = "aas.function.method";
+
+        internal const string AzureFunctionClass = "aas.function.class";
```

Comment:
Do these fall under AAS (from a user's point of view), or should they be their own top-level thing?

Reply:
These fall under AAS, at least as things stand now. 
We will only support functions on app service plans, until we figure out how to support consumption style, which is technically out of the scope of azure app service.

Reply:
I was thinking it may be confusing for users to see "AAS Function" as the tag name instead of "Azure Function". Also, if we ever support non-aas functions, wouldn't we want to use the same tag names?

Reply:
I talked with @steveharrington1 about this and the functions are all listed under App Services in the UI, including the consumption style. We're happy with where this nests.